site/src/components/TableLoader/TableLoader.tsx

```diff
@@ -22,3 +24,36 @@ const useStyles = makeStyles((theme) => ({
     height: theme.spacing(20),
   },
 }))
+
+export const TableLoaderSkeleton: FC<{
+  columns: number
+  rows?: number
+  useAvatarData?: boolean
+}> = ({ columns, rows = 4, useAvatarData = false }) => {
+  const placeholderColumns = Array(columns).fill(undefined)
+  const placeholderRows = Array(rows).fill(undefined)
+
+  return (
+    <>
```

Comment:
Should we add an accessibility attribute to indicate this is loading component, e.g. a `progressbar` role, or does MUI bake one into their `skeleton` component?

Reply:
Good question, I will take a look into that.